### ISSUE TITLE
Expose optimization of media transformations for alternative priors specification

### ISSUE THREAD

**Issue by wd60622** [pymc-labs/pymc-marketing#1038]
The media transformation are defined in pytensor could be leveraged in a function optimization in order to get parameters from more intuitive statements. For instance,

"At (scaled) spend of 1, I will have a contribution of 0.25"

```python
saturation = LogisticSaturation()

# Some lam and beta values
most_likely_parameters = saturation.desired_output_at_value(spend=1, desired_output=0.25)
```

I think something like this could be possible

What do marketer intuition tend to look like?

**Comment by sonriks6** [comment]
This is exactly what I need!

**Comment by wd60622** [comment]
> This is exactly what I need!

Hi @sonriks6 
Thanks for the feedback. Are there any other media spend spends that resonate well that might be missing?

**Comment by sonriks6** [comment]
Yes, we look at the ROI level expected regarding the net revenue and the media driven sales, which by default are estimated to contribute 25% (that's the marketers intuition)
